### PR TITLE
fix: fix import conversation with attachments to show one dialog for all items (Issue #1237)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
**Description:**

- Fixed import conversation with attachments to show one dialog for all items.
- Update Redux logic and epics.

Issues:

- Issue #1237 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1810.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1810.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1810.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — DISMISSED** [review]

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1810.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/store/import-export/importExport.epics.ts [hunk 1]
```diff
@@ -1135,72 +1206,174 @@ const uploadAllAttachmentsSuccessEpic: AppEpic = (action$, state$) =>
...
+                translate(
+                  successMessages.importAttachmentsSuccessConversationIgnored,
+                ),
+              ),
+            ),
+            of(ImportExportActions.resetState()),
+          );
+        }
+
+        // const updatedMessages: Message[] = conversationToUpload.messages.map(
```
Comment (IlyaBondar) [inline apps/chat/src/store/import-export/importExport.epics.ts]: Why do we need this commented code?

File: apps/chat/src/utils/app/import-export.ts [hunk 2]
```diff
@@ -293,29 +309,29 @@ export const updateAttachment = ({
...
-  if (!newAttachmentFile || !newAttachmentFile.name) {
-    return oldAttachment;
-  }
-
+  const newAttachmentUrl =
+    (oldAttachment.url || oldAttachment.reference_url) &&
+    (newAttachmentFile
+      ? constructPath(newAttachmentFile.absolutePath, newAttachmentFile.name)
+      : constructPath(
+          getFileRootId(BucketService.getBucket()),
```
Comment (IlyaBondar) [inline apps/chat/src/utils/app/import-export.ts]: you don't need to provide bucket here
```suggestion
          getFileRootId(),
```
also you can just use `getRootId()`

Comment (denys-kolomiitsev) [reply apps/chat/src/utils/app/import-export.ts]: Thank you!
Changed.

File: apps/chat/src/store/import-export/importExport.epics.ts (hunk 1 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/store/import-export/importExport.epics.ts]: We don't need this code.
Removed.

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1810.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**